### DISCUSSION TITLE
how to make sidebar?

### DISCUSSION THREAD

**Discussion by qqkkwan** [zauberzeug/nicegui#419] [Q&A] [answered]

**Comment by rodja** [comment, marked as answer]
Use `ui.left_drawer` or `ui.right_drawer`. See https://nicegui.io/reference#page_layout.

**Comment by qqkkwan** [comment]
Yes, i find the ui.left_drawer, but i dont't know what to do next to make a beautiful sidebar, are there any examples?

**Comment by rodja** [comment]
What is your definition of "beautiful"? We have a simple example at https://github.com/zauberzeug/nicegui/tree/main/examples/menu_and_tabs

**Reply by qqkkwan** [reply]
Sorry for my expression，I really like this project, but when it comes to the front end, I'm a bit lost. So is there any plan to provide a template, maybe NiceGUI Pro with build-in routes，sidebars and more，something like [antd design pro](https://preview.pro.ant.design/)

**Reply by rodja** [reply]
Thanks for the interesting suggestion. We have some open discussion about simplifying styling #117 where we template mechanisms are part of the picture. But this will still take some time figuring out.
